config/routes.rb

```diff
@@ -378,6 +378,8 @@
   post 'comment/react/delete/:id' => 'comment#react_delete'
   post 'comment/react/update/:id' => 'comment#react_update'
 
+  get 'switch_on_translation' => 'home#switch_on_translation'
```

Comment:
Yes, if possible if we could use existing routes, we'd be able to skip some of the tests because we wouldn't be introducing a whole new route, you know? That would keep codebase complexity down a bit, which is really helpful in the long run on a big project like this.